Formula/portable-libxcrypt.rb

```diff
@@ -0,0 +1,52 @@
+require File.expand_path("../Abstract/portable-formula", __dir__)
+
+class PortableLibxcrypt < PortableFormula
+  desc "Extended crypt library for descrypt, md5crypt, bcrypt, and others"
+  homepage "https://github.com/besser82/libxcrypt"
+  url "https://github.com/besser82/libxcrypt/releases/download/v4.4.28/libxcrypt-4.4.28.tar.xz"
+  sha256 "9e936811f9fad11dbca33ca19bd97c55c52eb3ca15901f27ade046cc79e69e87"
+  license "LGPL-2.1-or-later"
+
+  def install
+    ENV["HOMEBREW_CC"] = "/usr/lib/gcc-4.9-backport/bin/gcc" if ENV["HOMEBREW_ON_DEBIAN7"].present?
```

Comment:
Can this get a comment explaining what's going on and why? It's a bit hacky, I agree, but seems fine for something statically linked until we migrate away from Wheezy.